tauri-app/src/lib/components/tables/OrderTradesListTable.svelte

```diff
@@ -83,41 +83,31 @@
     </TableBodyCell>
     <TableBodyCell tdClass="break-all py-2">
       {formatUnits(
-        BigInt(item.outputVaultBalanceChange.amount),
+        BigInt(item.outputVaultBalanceChange.amount) * BigInt(-1),
         Number(item.outputVaultBalanceChange.vault.token.decimals ?? 0),
       )}
       {item.outputVaultBalanceChange.vault.token.symbol}
     </TableBodyCell>
     <TableBodyCell tdClass="break-all py-2" data-testid="io-ratio">
-      {Math.abs(
-        Number(
-          formatUnits(
-            BigInt(item.inputVaultBalanceChange.amount),
-            Number(item.inputVaultBalanceChange.vault.token.decimals ?? 0),
-          ),
-        ) /
-          Number(
-            formatUnits(
-              BigInt(item.outputVaultBalanceChange.amount),
-              Number(item.outputVaultBalanceChange.vault.token.decimals ?? 0),
-            ),
-          ),
+      {formatUnits(
+        (BigInt(10 ** 18) *
```

Comment:
this should probably be abstracted in the same way that it is onchain

e.g. `a.mul(b)` is like `(a * b) / ONE` and `a.div(b)` is `(a * ONE) / b`

nobody writes solidity like this, for a reason :)